receptor-python-worker/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = receptor-python-worker
+name = receptor_python_worker
```

Comment:
Why do you want to change it? It's usually good to keep the project name separate from the importable name.